.gitignore

```diff
@@ -0,0 +1,5 @@
+# dependencies
+/node_modules
+
+# vscode
+.vscode/*
```

Comment:
`.vscode/*` should be in your global gitignore, instead of project gitignore, since it's generated by personal tools as opposed to by the project.

(personally I'll be removing node_modules from my global gitignore since that caused me to not realise that this project wasn't ignoring node_modules...)